docker_files/etc/confd/templates/config.json.tmpl

```diff
@@ -0,0 +1,142 @@
+{
+	"wsPort": {{getv "/lisk/wsport" "8001"}},
+	"httpPort": {{getv "/lisk/httpport" "8000"}},
+	"address": "{{getv "/lisk/address" "0.0.0.0"}}",
+	"version": "{{getenv "LISK_VERSION"}}",
+	"minVersion": "{{getenv "LISK_MIN_VERSION"}}",
+	"fileLogLevel": "{{getv "/lisk/fileloglevel" "info"}}",
+	"logFileName": "{{getv "/lisk/logfilename" "logs/lisk.log"}}",
+	"consoleLogLevel": "{{getv "/lisk/consoleloglevel" "error"}}",
+	"trustProxy": {{getv "/lisk/trustproxy" "false"}},
+	"topAccounts": {{getv "/lisk/topaccounts" "false"}},
+	"cacheEnabled": {{getv "/lisk/cacheenabled" "false"}},
+	"wsWorkers": {{getv "/lisk/wsworkers" "1"}},
+	"db": {
+		"host": "{{getv "/lisk/db/host" "localhost"}}",
+		"port": {{getv "/lisk/db/port" "5432"}},
+		"database": "{{getv "/lisk/db/database" "lisk_main"}}",
+		"user": "{{getv "/lisk/db/user" "lisk"}}",
+		"password": "{{getv "/lisk/db/password" "password"}}",
+		"min": {{getv "/lisk/db/min" "10"}},
+		"max": {{getv "/lisk/db/max" "95"}},
+		"poolIdleTimeout": {{getv "/lisk/db/poolidletimeout" "30000"}},
+		"reapIntervalMillis": {{getv "/lisk/db/reapintervalmillis" "1000"}},
+		"logEvents": ["error"],
+		"logFileName": "{{getv "/lisk/db/logfilename" "logs/lisk_db.log"}}"
+	},
+	"redis": {
+		"host": "{{getv "/lisk/redis/host" "127.0.0.1"}}",
+		"port": {{getv "/lisk/redis/port" "6380"}},
+		"db": {{getv "/lisk/redis/db" "0"}},
+		"password": {{getv "/lisk/redis/password" "null"}}
+	},
+	"api": {
+		"enabled": {{getv "/lisk/api/enabled" "true"}},
+		"access": {
+			"public": {{getv "/lisk/api/access/public" "false"}},
+			{{$whiteListEntries := getvs "/lisk/api/access/whitelist/*"}}"whiteList": [{{range $index, $element := $whiteListEntries}}{{if $index}}, {{end}}"{{$element}}"{{end}}]
+		},
+		"options": {
+			"limits": {
+				"max": {{getv "/lisk/api/options/limits/max" "0"}},
+				"delayMs": {{getv "/lisk/api/options/limits/delayms" "0"}},
+				"delayAfter": {{getv "/lisk/api/options/limits/delayafter" "0"}},
+				"windowMs": {{getv "/lisk/api/options/limits/windowms" "60000"}}
+			},
+			"cors": {
+				"origin": "{{getv "/lil/api/options/cors/origin" "*"}}",
+				"methods": ["GET", "POST", "PUT"]
+			}
+		}
+	},
+	"peers": {
+		"enabled": {{getv "/lisk/peers/enabled" "true"}},
+		"list": [
```

Comment:
This is the right use case for configurable values. User may want load its own list of peers. Can't understand why its should be done separately. Issue #398 is totally different thing, to support multiple networks in one installation.